README.md

```diff
@@ -26,29 +26,31 @@
 </p>
 
 ## **Table of contents**
-1. [**What is Horusec?**](#What-is-Horusec?)
-2. [**Getting started**](#Getting-started)
-    1. [**Requirements**](#Requirements)
-    2. [**Installing Horusec**](#Installing-Horusec)
-    3. [**Usage**](#Usage)
-        1. [**CLI Usage**](#CLI-Usage)
-        2. [**Using Docker**](#Using-Docker)
-        3. [**Older versions**](#Older-versions)
-        4. [**Using Horusec-Web application**](#Using-Horusec-Web-application)
-        5. [**Using Visual Studio Code**](#Using-Visual-Studio-Code)
-        6. [**Using the Pipeline**](#Using-the-Pipeline)
-4. [**Contributing**](#Contributing)
-5. [**Roadmap**](#Community)
-6. [**License**](#License)
-7. [**Community**](#Community)
+### 1. [**About**](#About)
+### 2. [**Getting started**](#Getting-started)
+>#### i.   [**Requirements**](#Requirements)
+>#### ii.  [**Installation**](#Installation)
```

Comment:
Should the usage assignment follow the numbering?

Example:

## **Table of contents**
### 1. [**About**](#About)
### 2. [**Getting started**](#Getting-started)
>#### i.   [**Requirements**](#Requirements)
>#### ii.  [**Installation**](#Installation)
>#### iii. [**Usage**](#Usage)
>##### i. [**CLI Usage**](#CLI-Usage)
>##### ii. [**Using Docker**](#Using-Docker)
>##### iii. [**Older versions**](#Older-versions)
>##### iv. [**Using Horusec-Web application**](#Using-Horusec-Web-application)
>##### v. [**Using Visual Studio Code**](#Using-Visual-Studio-Code)
>##### vi. [**Using the Pipeline**](#Using-the-Pipeline)
### 3. [**Documentation**](#Documentation)       
### 4. [**Roadmap**](#Roadmap)
### 5. [**Contributing**](#Contributing)
### 6. [**Code of Conduct**](#Code-of-Conduct)
### 7. [**License**](#License)
### 8. [**Community**](#Community)

Reply:
I did that, but the link wasn't working, so I removed it. I will try again.